### ISSUE TITLE
Verify use of ObjectMapper

### ISSUE THREAD

**Issue by petrdvorak** [wultra/powerauth-server#531]
We are often using a new instance of ObjectMapper, like so:

```java
private final ObjectMapper objectMapper = new ObjectMapper();
```

We should check if we can configure ObjectMapper globally.

**Comment by romanstrobl** [comment]
Good point. We should also evaluate the ObjectMapper configuration. For instance, in ECIES we have a problem with compatibility between versions when new fields are not recognized, which leads to hard to diagnose errors. Sometimes the object mapper needs to be strict and sometimes it's strictness is counter-productive. We should evaluate each usage separately.

**Comment by romanstrobl** [comment]
See: https://github.com/wultra/powerauth-server/issues/446

**Comment by romanstrobl** [comment]
Fixed in https://github.com/wultra/powerauth-server/pull/2372
